package.json

```diff
@@ -22,16 +22,18 @@
   ],
   "main": "lib/index.js",
   "scripts": {
-    "test": "mocha"
+    "test": "standard && dependency-check ."
```

Comment:
You should keep the mocha testing here, like `standard && dependency-check . && mocha`